Gems/Multiplayer/Code/Source/Components/LocalPredictionPlayerInputComponent.cpp

```diff
@@ -275,17 +275,21 @@ namespace Multiplayer
 
                 correction.Resize(serializer.GetSize());
 
-                AZLOG_INFO("** Autonomous Desync - Corrected clientInputId=%d at hostFrame=%u hostTime=%lu", aznumeric_cast<int32_t>(m_lastClientInputId),
-                    aznumeric_cast<uint32_t>(m_lastInputReceived[0].GetHostFrameId()), aznumeric_cast<uint64_t>(m_lastInputReceived[0].GetHostTimeMs()));
+                AZLOG_INFO(
+                    "** Autonomous Desync - Corrected clientInputId=%hu at hostFrame=%u hostTime=%" PRId64,
+                        static_cast<uint16_t>(m_lastClientInputId),
```

Comment:
cannot keep the safety of _numeric_cast_ after using _PRId64_? _PRId64 only applies to the 64bit number (right?), so could we keep numeric_cast on the others?

Reply:
Switching to static actually has nothing to do with it. It just didn't make sense to numeric cast here in the first place.